homeassistant/components/media_player/squeezebox.py

```diff
@@ -201,11 +201,18 @@ def media_duration(self):
     def media_image_url(self):
         """ Image url of current playing media. """
         if 'artwork_url' in self._status:
-            return self._status['artwork_url']
-        return ('http://{server}:{port}/music/current/cover.jpg?'
-                'player={player}').format(server=self._lms.host,
-                                          port=self._lms.http_port,
-                                          player=self._id)
+            media_url = self._status['artwork_url']
+        else:
+            media_url = ('/music/current/cover.jpg?'
+                         'player={player}&'
+                         'nocache={nocache}').format(
+                             player=self._id,
+                             nocache=hash(self.media_title))
```

Comment:
https://crestron-squeeze.googlecode.com/files/cli-api_7.6.htm this is the docs I used to figure out how it works. 
According to the docs the status command should return a "Track Id". I don't know how this Id is composed or if it is unique or not. But might be worth a try. Might already be available in ```self._status ```